posthog/queries/funnels/base.py

```diff
@@ -711,7 +711,11 @@ def _get_breakdown_select_prop(self) -> str:
                 )
         elif self._filter.breakdown_type == "event":
             basic_prop_selector = get_single_or_multi_property_string_expr(
-                self._filter.breakdown, table="events", query_alias="prop_basic", column="properties"
+                self._filter.breakdown,
```

Comment:
q: Should normalising apply to person properties as well?

For example, when breaking down by `$initial_current_url` ? I think it should?

Reply:
which means L700 too

Reply:
(but this might be scope screep so treat as nit)

Reply:
Hmmm, it's a reasonable follow-up though. I'll put it as a follow-up though. I don't want to accidentally run down the clock and miss getting this fix in the release